packages/core/src/components/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import {
+  Pressable,
+  StyleSheet,
+  View,
+  StyleProp,
+  ViewStyle,
+} from "react-native";
+
+import Text from "../Text";
+import type { IconSlot } from "../../interfaces/Icon";
+import {
+  COMPONENT_TYPES,
+  createIconProp,
+  createTextProp,
+} from "@draftbit/types";
+import { extractStyles } from "../../utilities";
+import { withTheme } from "../../theming";
+import type { Theme } from "../../styles/DefaultTheme";
+
+type Props = {
+  icon?: string;
+  label: string;
+  iconColor?: string;
+  style: StyleProp<ViewStyle>;
```

Comment:
```suggestion
  style?: StyleProp<ViewStyle>;
```
(to match AccordionGroup)